Pseudocode/Dynamic Programming/Fibonacci/SourceCode.tex

```diff
@@ -0,0 +1,62 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+% Define the command for a boldface instructions
+\newcommand{\Is}{\textbf{ is }}
+\newcommand{\To}{\textbf{ to }}
+\newcommand{\Downto}{\textbf{ downto }}
+\newcommand{\Or}{\textbf{ or }}
+\newcommand{\And}{\textbf{ and }}
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Calculate N-th term of the fibonacci sequence using a iterative dynamic programming approach (bottom up).}
```

Comment:
`\caption{Calculate N-th term of the Fibonacci sequence using iterative DP (Bottom Up)}`

(This is to make the precise statement in a single line instead of 2 lines.